### PR TITLE
Css refactor

### PR DESCRIPTION
**Pull request by felicia-goh**
- refactored css to remove "style=.." from the html
- removed unused css classes
- tried to rename classes descriptively where possible

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,34 +1,37 @@
...
-  post '/payslips/send', to: 'payslips#send_payslip'
-  post '/users/send', to: 'users#send_ipa'
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   resources :accidents do
     member do
       get :new3
       get :complete
     end
   end
+  get '/accidents/confirm', to: 'accidents#confirm'
```
Comment (christian-quah) [inline config/routes.rb:16]: I think this is the one that causes the confirm to not work

**Review by christian-quah — COMMENTED** [review]
I think it takes confirm as a :id param like this.

File: config/routes.rb (hunk 1 above)
Comment (felicia-goh) [reply config/routes.rb:16]: Ohh ic, thanks for finding the specific one!

**Review by christian-quah — APPROVED** [review]
Looks good!

**Pull request merged**